src/internal/types.ts

```diff
@@ -101,3 +101,5 @@ export interface SchedulerAction<T> extends Subscription {
 export type ObservedValueOf<O> = O extends ObservableInput<infer T> ? T : never;
 
 export type ObservedValuesFromArray<X> = X extends Array<ObservableInput<infer T>> ? T : never;
+
+export type ArrayValueOf<A> = A extends Array<infer T> ? T : never;
```

Comment:
Given that the above type is named `ObservedValuesFromArray` and extracts the observed values from an array and this one extracts the value from an array, should it be called `ValueFromArray`?